composer.json

```
@@ -1,15 +1,15 @@
 {
-    "name": "vaimo/webdriver-binary-downloader",
+    "name": "bolt/webdriver-binary-downloader",
```

```suggestion
    "name": "lanfest/webdriver-binary-downloader",
```

But I suppose you were already planning to change the namespaces. :-)